Makefile.inc

```diff
@@ -54,6 +54,15 @@ skipclippy:
 
 # Testing. A lot of directories can't test either, yet.
 test:
+	cargo test
 
 skiptest:
 	echo Not doing test for this target
+
+coverage:
+	cargo tarpaulin --out Lcov --output-dir "$(OREBOOT)/coverall"
+	cat "$(OREBOOT)/coverall/lcov.info" >> $(OREBOOT)/coverall/lcov.temps
+	rm "$(OREBOOT)/coverall/lcov.info"
+
+skipcoverage:
```

Comment:
this is not necessary; either a test can run, then it can also be run with coverage, or skip it, using `skiptest`

Reply:
remove this